src/Breadcrumb/EzContentBreadcrumbBuilder.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Drupal\ezcontent_api\Breadcrumb;
+
+use Drupal\Core\Breadcrumb\Breadcrumb;
+use Drupal\Core\Breadcrumb\BreadcrumbBuilderInterface;
+use Drupal\Core\Routing\RouteMatchInterface;
+use Drupal\Core\Link;
+
+/**
+ * Implements custom breadcrumb builder.
+ */
+class EzContentBreadcrumbBuilder implements BreadcrumbBuilderInterface {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function applies(RouteMatchInterface $attributes) {
+    return TRUE;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function build(RouteMatchInterface $route_match) {
+    $entity = $route_match->getParameter('entity');
+    $links = [];
+    if ($entity) {
+      $links[] = Link::createFromRoute(t('Home'), '<front>');
+      $links[] = $entity->bundle();
```

Comment:
Will this not provide the machine name ?
If yes, we would need the human readable name, which can be fetched via `$entity->type->entity->label()`;